the-graph/subgraph.yaml

```
@@ -91,6 +91,27 @@ templates:
         - event: TokenDeposited(indexed address,indexed address,uint256)
           handler: habdleTokenDeposited
     network: localhost
+  - name: CashRemunerationEIP712
+    kind: ethereum/contract
+    source:
+      abi: CashRemunerationEIP712
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.9
+      language: wasm/assemblyscript
+      file: ./src/cashRemunerationEip712.ts
+      entities:
+        - Withdraw
+        - Deposited
```

```suggestion
        - Transaction
```

You need to use `Transaction` since that's the one that we're using in the handler for transfers, deposits and withdrawals. I need to update mine as well